test/pylib/minio_server.py

```diff
@@ -38,8 +41,41 @@ def __init__(self, tempdir_base, hosts, logger):
         self.default_user = 'minioadmin'
         self.default_pass = 'minioadmin'
         self.bucket_name = 'testbucket'
+        self.access_key = ''.join(random.choice(string.hexdigits) for i in range(16))
+        self.secret_key = ''.join(random.choice(string.hexdigits) for i in range(32))
         self.log_filename = (self.tempdir / 'minio').with_suffix(".log")
 
+    def bucket_policy(self):
+        return {
+              'Version': '2012-10-17',
+              'Statement': [
+                {
+                  'Action': [
+                    's3:ListBucket'
```

Comment:
nice! i just blindly used the public anonymous policy instead of cherry-picking the ones we're actually using.